### PR TITLE
Implement Personal Messages with Execution

### LINKED ISSUE #10: As a long-time user, I want personalized messages from the app after executing tasks.

### PR DESCRIPTION
**Pull request by yAOwzers**
Plan to store the name of user

### CODE REVIEW

**Comment by NeilBaner** [comment]
> Plan to store the name of user

How do you think we should do this? Perhaps modify the Storage class to save a file like `preferences.txt` or something?

**Comment by yAOwzers** [comment]
Able to save the name of the user in the nameOfUser.txt file, however during the first iteration does not save the name immediately. Still currently working on it

**Comment by yAOwzers** [comment]
Do y'all think it's necessary to include all the isPersonalised() and isTriggerAutoSave() methods?

**Comment by yAOwzers** [comment]
The IO test will fail now since there's a welcome message right? Or is there something wrong with my code HMM

**Comment by franceneee** [comment]
> The IO test will fail now since there's a welcome message right? Or is there something wrong with my code HMM

I don't think it has anything to do with the IO part? 
image

I updated the IO part after adding the welcome bit... haha maybe you're missing an import statement somewhere?

**Comment by NeilBaner** [comment]
Marked as draft since it doesn't seem to be done quite yet.

**Comment by yAOwzers** [comment]
OOOOOOOORight it was merge conflicts... lol

**Comment by yAOwzers** [comment]
Ready for review

**Review by Lusi711 — APPROVED** [review]

File: src/main/resources/txt/personalMessages.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Sensational
+I like how you think
+Good remembering
+You know just what to do
+You expressed yourself so well
+You are a creative thinker
+Yes sure
+Wow sure thing
+Ok will do
+You just know what to do
```
Comment (franceneee) [inline src/main/resources/txt/personalMessages.txt:14]: HAHAHA NICE.

**Review by franceneee — APPROVED** [review]

**Pull request merged**